cmd/neofs-node/config.go

```diff
@@ -219,6 +220,7 @@ func (a *applicationConfiguration) readConfig(c *config.Config) error {
 			wc.smallObjectSize = writeCacheCfg.SmallObjectSize()
 			wc.flushWorkerCount = writeCacheCfg.WorkersNumber()
 			wc.sizeLimit = writeCacheCfg.SizeLimit()
+			wc.noSync = writeCacheCfg.BoltDB().NoSync()
```

Comment:
bolt config that could not be applied to bolt?

Reply:
Writecache has a flat struct, this is just `writecache.no_sync`

Reply:
yes, i understand, just confusing a little from the code side, not user's